bazel/core.bzl

```diff
@@ -57,6 +57,7 @@ MLN_CORE_SOURCE = [
     "src/mbgl/gfx/shader_registry.cpp",
     "src/mbgl/gfx/shader_group.cpp",
     "src/mbgl/gfx/stencil_mode.hpp",
+	"src/mbgl/gfx/symbol_drawable_data.hpp",
```

Comment:
You include this file in the core here but in the list of drawables source for CMakeLists. Move this to MLN_DRAWABLES_HEADERS. Everything else looks good.